ibridges/__main__.py

```diff
@@ -155,6 +162,28 @@ def ibridges_list():
     with interactive_auth(irods_env_path=_get_ienv_path()) as session:
         _list_coll(session, _parse_remote(args.remote_path, session))
 
+def _create_coll(session: Session, remote_path: IrodsPath):
+    if remote_path.exists():
+        raise ValueError(f'New collection path {remote_path} already exists.')
+    else:
+        remote_path.create_collection(session, remote_path)
```

Comment:
Let's just put this inline (not with it's own function), since it's only so few lines.